spec/features/guide_history_spec.rb

```diff
@@ -1,9 +1,18 @@
 require 'rails_helper'
 
 RSpec.describe "Guide history", type: :feature do
+  include ActiveSupport::Testing::TimeHelpers
+
+  after do
+    travel_back
+  end
+
   it "shows a history of the latest edition" do
     stub_publisher
     create_guide_community
+
+    travel_to Time.new(2004, 11, 24, 01, 04, 44)
```

Comment:
This guy can take a block which saves using the `after` callback. I think it reads nicer using the block form because you don't have to look outside the `it` block when trying to work out what's going on..